api-server/src/server/utils/publicUserProps.js

```diff
@@ -25,6 +25,7 @@ export const publicUserProps = [
   'isQaCertV7',
   'isInfosecCertV7',
   'isJsAlgoDataStructCert',
+  'isRelationalDatabasesCertV8',
```

Comment:
I cannot see this file being used anywhere.

Reply:
It's used here: https://github.com/freeCodeCamp/freeCodeCamp/blob/57cf47dad4f1d94b2cae6dba7dfd237dc09c8e46/api-server/src/common/models/user.js#L29